site/releases/2024-sep-09/release-notes.qmd

```diff
@@ -0,0 +1,138 @@
+Labels: enhancement
+--->
+
+### Reset your model workflow
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+
+Need to start over with a model? You can now reset your model's workflow to its initial state in the lifecycle.
+
+<form method="get" action="/guide/model-workflows/working-with-model-workflows.html#reset-your-model-workflow" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">Working with model workflows</button>
+</form>
+
+:::
+
+::: {.w-30-ns}
+
+![](reset-model-workflow.png){fig-alt="A screenshot showing the option to reset your model workflow"}
+
+
+:::
+
+::::
+
+## Enhancements
+
+### Model inventory fields
+
+Previously, we used custom fields to refer to your organization's unique additional fields for models as well as the default fields provided as suggestions by ValidMind. Custom fields has been renamed to model inventory fields, capturing all additional fields, and the terminology standardized across the {{< var validmind.platform >}} and our documentation.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+You can still require additional fields to be provided upon model registration, and work with a number of different types of fields.
+
+:::
+
+::: {.w-30-ns}
+
+<form method="get" action="/guide/model-inventory/manage-model-inventory-fields.html" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">Manage model inventory fields</button>
+</form>
+
+:::
+
+::::
+
+
+
+<!---
+PR #929: Renamed Statuses to Model Lifecycle Statuses
+URL: https://github.com/validmind/frontend/pull/929
+Labels: enhancement
+--->
+### Model lifecyle statuses
+
+Previously, we resource statuses to refer to the model stages made available for use in your model workflows. Resource statuses has been renamed to model lifecycle statuses, clarifying their function, and the terminology standardized across the {{< var validmind.platform >}} and our documentation.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+- Default statuses are provided for you as suggestions. 
+- You can add, edit, remove, and define your own custom statuses.
+
+:::
+
+::: {.w-30-ns}
+
+<form method="get" action="/guide/model-workflows/customize-model-lifecycle-statuses.html" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">Customize model lifecycle statuses</button>
+</form>
+
+:::
+
+::::
+
+
+## Bug fixes
+
+<!---
+PR #910: fix template editor showing multiorg roles
+URL: https://github.com/validmind/frontend/pull/910
+Labels: bug
+--->
+
+### Multi-organizational roles display issue 
```

Comment:
I wonder if this needs to be included. I thought it was more of an "internal" feature, and customers (as of now) wouldn't belong to more than one organization.